### PR TITLE
Add back MFR integration

### LINKED ISSUE #13801: [ProjectBlue] Crash on loading MFR integration
### Your GTNH Discord Username

_No response_

### Your Pack Version

Not relevant

### Your Server

SP

### Java Version

Java 8

### Type of Server

Single Player

### Your Expectation

No crash?…

### The Reality

```
A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Stacktrace:
	at gcewing.projectblue.BaseMod.newIntegration(BaseMod.java:207)
	at gcewing.projectblue.BaseMod.integrateWith(BaseMod.java:196)
	at gcewing.projectblue.ProjectBlue.preInit(ProjectBlue.java:70)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at cpw.mods.fml.common.FMLModContainer.handleModStateEvent(FMLModContainer.java:532)
	at sun.reflect.GeneratedMethodAccessor2.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at com.google.common.eventbus.EventSubscriber.handleEvent(EventSubscriber.java:74)
	at com.google.common.eventbus.SynchronizedEventSubscriber.handleEvent(SynchronizedEventSubscriber.java:47)
	at com.google.common.eventbus.EventBus.dispatch(EventBus.java:322)
	at com.google.common.eventbus.EventBus.dispatchQueuedEvents(EventBus.java:304)
	at com.google.common.eventbus.EventBus.post(EventBus.java:275)
	at cpw.mods.fml.common.LoadController.sendEventToModContainer(LoadController.java:212)
	at cpw.mods.fml.common.LoadController.propogateStateMessage(LoadController.java:190)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at com.google.common.eventbus.EventSubscriber.handleEvent(EventSubscriber.java:74)
	at com.google.common.eventbus.SynchronizedEventSubscriber.handleEvent(SynchronizedEventSubscriber.java:47)
	at com.google.common.eventbus.EventBus.dispatch(EventBus.java:322)
	at com.google.common.eventbus.EventBus.dispatchQueuedEvents(EventBus.java:304)
	at com.google.common.eventbus.EventBus.post(EventBus.java:275)
	at cpw.mods.fml.common.LoadController.distributeStateMessage(LoadController.java:119)
	at cpw.mods.fml.common.Loader.preinitializeMods(Loader.java:556)
	at cpw.mods.fml.client.FMLClientHandler.beginMinecraftLoading(FMLClientHandler.java:243)
	at net.minecraft.client.Minecraft.func_71384_a(Minecraft.java:480)

-- Initialization --
Details:
Stacktrace:
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:878)
	at net.minecraft.client.main.Main.main(SourceFile:148)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28)
```

### Your Proposal

Crashes on both 1.1.7 and 1.1.8 releases.

[2023-06-25-7.log](https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/files/11860579/2023-06-25-7.log)

### Final Checklist

- [X] I have searched this issue tracker and there is nothing similar already. Posting on a closed issue saying the bug still exists will prompt us to investigate and reopen it once we confirm your report.
- [X] I can reproduce this problem consistently by follow the exact steps I described above, or this does not need reproducing, e.g. recipe loophole.
- [X] I have asked other people and they confirm they also have this problem by follow the exact steps I described above, or this does not need reproducing, e.g. recipe loophole.

### PR DESCRIPTION
**Pull request by miozune**
Closes https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/13801

### CODE REVIEW

**Comment by wlhlm** [comment]
@AnrDaemon Can you please confirm this fixes the crash for you? You can download a pre-built jar from here: https://github.com/GTNewHorizons/ProjectBlue/actions/runs/5387845819 (build-libs)

**Comment by AnrDaemon** [comment]
Sorry for the late reply, had no access to MC capable system.
Yes, that seems to have fixed the ProjectBlue issue.

**Review by wlhlm — APPROVED** [review]

File: src/main/scala/gcewing/projectblue/mfr/MFRIntegration.java [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+package gcewing.projectblue.mfr;
+
+import net.minecraft.block.*;
+import net.minecraft.init.*;
+import net.minecraft.item.*;
+
+import cpw.mods.fml.common.registry.*;
+import gcewing.projectblue.*;
+
+public class MFRIntegration extends gcewing.projectblue.BaseIntegration {
```
Comment (eigenraven) [inline src/main/scala/gcewing/projectblue/mfr/MFRIntegration.java:16]: Does anything actually initialize this class? I can't see references to it in this pr

Comment (wlhlm) [reply src/main/scala/gcewing/projectblue/mfr/MFRIntegration.java:16]: Looks like it loads the integration dynamically:

https://github.com/GTNewHorizons/ProjectBlue/blob/c8a5dfaf3197aa074e20b6255666d35d94baaf88/src/main/scala/gcewing/projectblue/ProjectBlue.java#L66-L71

**Pull request merged**